### PR TITLE
Cleanup

### LINKED ISSUE #27: Incorporate color coding of background color of the chore cards for each member!
Maybe we can assign a specific color for each user or each chore and store the hex code in the BE so when we fetch the chores we can pull that code and use it for a more cohesive feel... Extension could be adding a color picker to the chore form to allow the user to decide the color!

### PR DESCRIPTION
**Pull request by Kanderson58**
# Description

- Remove console.log
- Refactor
- Syntax check
- Style changes
- Loading changes

## Type Of Change

- [ ]  Bug fix
- [ ]  New feature
- [x]  Refactoring
- [ ]  Testing

# How Has This Been Tested?

- [x]  local host
- [ ]  dev tools
- [x]  cypress

# Related Issues:

- [x]  closes  #27 

# Checklist:

- [x]  My code follows the Turing's guidelines of this project
- [x]  I have performed a self-review of my own code
- [x]  No console error messages

### CODE REVIEW

File: src/Components/ChoreCard/ChoreCard.js
Comment (LeftyLincoln) [inline src/Components/ChoreCard/ChoreCard.js:1]: This is nifty code

**Review by LeftyLincoln — APPROVED** [review]
LGTM

File: src/Components/HouseForm/HouseForm.js [hunk 1]
```diff
@@ -155,7 +158,7 @@ const HouseForm = ({ email }) => {
           </form>
         )}
         {memberInputs}
-        {!editMember && (
+        {!editMember && members.length < 10 && (
```
Comment (qrispi) [inline src/Components/HouseForm/HouseForm.js:161]: Love this!

File: src/Components/ChoreCard/ChoreCard.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import './ChoreCard.css';
 
-const ChoreCard = ({member, chore, duration}) => {
+const ChoreCard = ({member, allMembers, chore, duration}) => {
   return (
-    <div className='chore-card' tabIndex="0" style={{'backgroundColor':'#3b425d'}}>
+    <div className='chore-card' tabIndex="0" style={{'backgroundColor':allMembers.find(singleMember => singleMember.name === member).color}}>
```
Comment (qrispi) [inline src/Components/ChoreCard/ChoreCard.js:5]: Thanks for showing us this!

**Review by qrispi — APPROVED** [review]
LGTM!

File: src/Components/ChoreCard/ChoreCard.css
Comment (LeftyLincoln) [inline src/Components/ChoreCard/ChoreCard.css:1]: I like this, makes the chore cards look clean

File: src/Components/Dashboard/Dashboard.js
Comment (LeftyLincoln) [inline src/Components/Dashboard/Dashboard.js:1]: I like setting the amount of colors to the amount of possible chore-doers

**Review by LeftyLincoln — APPROVED** [review]
LGTM

**Pull request merged**